File: Natom.Gestion.WebApp.Clientes.Frontend/src/app/classes/dto/app.config.dto.ts
import { NegocioConfigDTO } from "./negocio/negocio.config.dto";
import { FeatureFlagsDTO } from "./_feature_flags/feature-flags.dto";

export class AppConfigDTO {
    public feature_flags: FeatureFlagsDTO;
    public negocio_config: NegocioConfigDTO;
}

File: Natom.Gestion.WebApp.Clientes.Backend.Entities/DTO/AppConfigDTO.cs
using Natom.Gestion.WebApp.Clientes.Backend.Entities.DTO.Negocio;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Natom.Gestion.WebApp.Clientes.Backend.Entities.DTO
{
    public class AppConfigDTO
    {
        [JsonProperty("feature_flags")]
        public FeatureFlagsDTO FeatureFlags { get; set; }

        [JsonProperty("negocio_config")]
        public NegocioConfigDTO NegocioConfig { get; set; }
    }
}
